### PR TITLE
Decrease xml test log size

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: scripts/tests.sh [hunk 1]
```diff
@@ -209,9 +209,9 @@ do
         then
         if [ -n "$optimize" ]
         then
-            log=--logger=JUNIT,test_suite,$log_directory/opt_$vm.xml $testargs
+            log=--log_level=test_suite --logger=JUNIT,test_suite,$log_directory/opt_$vm.xml $testargs
```
Comment (chriseth) [inline scripts/tests.sh]: Isn't that what we already have? Also, I think this should activate logging on stdout in addition to logging to the xml file. The docs claim that the level of the junit logger cannot be changed: https://www.boost.org/doc/libs/1_68_0/libs/test/doc/html/boost_test/test_output/log_formats/log_junit_format.html

Comment (chriseth) [reply scripts/tests.sh]: Also: "Until Boost 1.64, the log level was previously defaulting to success and was causing a heavy load on the logging part in some circumstances. "

Comment (Marenz) [reply scripts/tests.sh]: Well, it does seem to break something:
```
scripts/tests.sh: line 214: --logger=JUNIT,test_suite,test_results/noopt_homestead.xml: No such file or directory
```

Comment (Marenz) [reply scripts/tests.sh]: Ah, probably too many spaces

Comment (Marenz) [reply scripts/tests.sh]: okay, I see what you mean by `Isn't that what we already have? `, I didn't check the --logger parameter format before

**Comment by Marenz** [comment]
This change seems to have decreased the size significantly. Should be good to go

**Review by chriseth — APPROVED** [review]

**Pull request merged**